honeycomb-template/src/client/client.js

```diff
@@ -1,8 +1,25 @@
 import React from 'react';
 import { render } from 'react-dom';
 import Example from './components/Example.jsx';
+import { AppContainer } from 'react-hot-loader';
+
+const appEl = document.querySelector('.js-app');
+const name = 'world';
 
 render(
-  <Example name="world" />,
-  document.querySelector('.js-app')
+  <AppContainer>
+    <Example name={name} />
+  </AppContainer>,
+  appEl
 );
+
+if (module.hot) {
+  module.hot.accept(() => {
```

Comment:
@chrishelgert I'm not sure if it's right, see https://github.com/gaearon/redux-devtools/blob/master/examples/todomvc/index.js#L19